drivers/scheduler/k8s/specs/elasticsearch/pure/pure-storage-class.yaml

```diff
@@ -0,0 +1,15 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+    name: elasticsearch-sc
+provisioner: kubernetes.io/portworx-volume
```

Comment:
Are we planning to run this using the portworx in-tree driver or the CSI driver?

Reply:
I think you're right, checking with @david-enli to be sure

Reply:
Confirmed, will go ahead and fix this